pkgs/development/python-modules/gmplot/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, requests
+}:
+
+buildPythonApplication rec {
```

```suggestion
buildPythonPackage rec {
```

If this is an actual application:

According to the readme, this is meant to also be used as an application. Do you mind adding the following to `pkgs/top-level/all-packages.nix`:
```
  <pkg-name> = with python3Packages; toPythonApplication <python-module>;
```
More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)